VERSION "HIPBNYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY/4/%%%/4/'%**4YYY///"


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: TestNode


BO_ 1024 Message1: 8 TestNode
 SG_ Signal1 : 0|64@1- (1,0) [0|0] "" Vector__XXX

BO_ 1025 Message2: 8 TestNode
 SG_ Signal2 : 32|32@1- (1,0) [0|0] "" Vector__XXX
 SG_ Signal1 : 0|32@1- (1,0) [0|0] "" Vector__XXX




CM_ BU_ TestNode "";
SIG_VALTYPE_ 1024 Signal1 : 2;
SIG_VALTYPE_ 1025 Signal1 : 1;
SIG_VALTYPE_ 1025 Signal2 : 1;





